### ISSUE TITLE
Inconsistency between redis clients

### ISSUE THREAD

**Issue by AMecea** [adjust/rmq#31]
Hello,

The [go redis client](https://github.com/go-redis/redis) recommends using package: `"github.com/go-redis/redis"`, but this project uses: `"gopkg.in/redis.v3"`.

Trying to use those two packages together I get the following error:
```
cannot use (type *"github.com/foo/bar/vendor/github.com/go-redis/redis".Client) as type *"github.com/foo/bar/vendor/gopkg.in/redis.v3".Client in argument to rmq.OpenConnectionWithRedisClient
```

I think this project dependencies should be updated to redis [v6](https://gopkg.in/redis.v6) and use `github.com/go-redis/redis`.

**Comment by wellle** [comment]
@AMecea: Thank you, I will have a look 👍

**Comment by wellle** [comment]
@AMecea: Thanks again for bringing this up. We just merged #32. Can you try again?
